src/IBM.WatsonDeveloperCloud/Constants.cs

```diff
@@ -26,6 +26,6 @@ public class Constants
         /// The version number for this SDK build. Added to the header in 
         /// each request as `User-Agent`.
         /// </summary>
-        public const string SDK_VERSION = "0.0.1-alpha";
+        public const string SDK_VERSION = "watson-apis-dotnet-standard-sdk/0.2.0-alpha";
```

Comment:
@atilatosta I am versioning this here - I probably should have done this in another branch but I wanted to keep in mind the standard format they would like the sdk version. Also we need to find a way to automatically version this on release.